### ISSUE TITLE
editor cannot find java.exe

### ISSUE THREAD

**Issue by PItlosTails** [goatfungus/NMSSaveEditor#324]
despite downloading both the offline and online version of java 8 that was linked the editor tells me that "The system cannot find the file C:\ProgramData\Oracle\Java\javapath\java.exe" what is the problem? and how would i go at fixing it? im pretty much coding illiterate so i dont know what the problem is

**Comment by Infinality** [comment]
Might try uninstalling all version of Java installed, and then install the Windows installer from here:   https://www.oracle.com/java/technologies/javase-jdk11-downloads.html

**Comment by jesper987654** [comment]
i am also having that problem... 

The system cannot find the file specified.
'java' is not recognized as an internal or external command,
operable program or batch file.

that is the only thing that shows in the console when trying to run it....
if it is looking for java.exe then it is in C:\Program\Java\jre1.8.0_261\bin\java.exe and in C:\Program Files\Java\jre1.8.0_261\bin\java.exe for me and not in oracle's folder...

Also if what you are sugesting with that link is that i should download jdk-11.0.8_windows-x64_bin.exe from oracle then there has to be a way around it because i am NOT creating an account on oracle just to launch a file... especially when it asks for personal information and information that i don't even have like job title and company name...

**Comment by Infinality** [comment]
Yeah, Oracle is evil like that.  I haven't tested it, but here's OpenJDK:   https://adoptopenjdk.net/

**Comment by jesper987654** [comment]
> Yeah, Oracle is evil like that. I haven't tested it, but here's OpenJDK: https://adoptopenjdk.net/

Thank you it works

**Comment by Infinality** [comment]
> > Yeah, Oracle is evil like that. I haven't tested it, but here's OpenJDK: https://adoptopenjdk.net/
> 
> Thank you it works

Glad to hear!
